asset-transfer-events/application-gateway-typescript/src/app.ts

```diff
@@ -134,8 +134,9 @@ async function deleteAssetByID(contract: Contract): Promise<void>{
     console.log('\n*** DeleteAsset committed successfully');
```

Comment:
likewise this typescript file?

Reply:
Yes, again This is the sample application.